.dagger/build/sdk.go

```diff
@@ -70,6 +70,10 @@ func (build *Builder) pythonSDKContent(ctx context.Context) (*sdkContent, error)
 }
 
 func (build *Builder) typescriptSDKContent(ctx context.Context) (*sdkContent, error) {
+	sdkCache := dag.Container().
+		From(consts.TypescriptImage).
+		WithExec([]string{"npm", "install", "-g", "tsx@4.15.6"})
```

Comment:
That's the same for  node_modules installation btw, you don't even have to care about the package manager you use in this sdkCache.
As long as you can get an optimized artifact to mount to the tarball, it's perfect :)